ansible_base/authentication/serializers/authenticator.py

```diff
@@ -95,10 +96,10 @@ def validate(self, data) -> dict:
             if configuration or configuration == {}:
                 for key in authenticator.configuration_encrypted_fields:
                     if not self.instance and configuration.get(key, None) == ENCRYPTED_STRING:
-                        invalid_encrypted_keys[key] = f"Can not be set to {ENCRYPTED_STRING}"
+                        invalid_encrypted_keys[key] = _("Can not be set to %(ENCRYPTED_STRING)") % {"ENCRYPTED_STRING": ENCRYPTED_STRING}
                 if invalid_encrypted_keys:
                     raise ValidationError(invalid_encrypted_keys)
                 data['configuration'] = authenticator.validate_configuration(configuration, self.instance)
             return data
         except ImportError as e:
-            raise ValidationError({'type': f'Failed to import {e}'})
+            raise ValidationError({'type': _('Failed to import %(e)') % {'e': e}})
```

Comment:
same as the others.

Reply:
@thedoubl3j For some reason GitHub won't let me resolve this conversation. Everything is green now! I didn't understand that the `s` is part of the pattern. I've updated my notes accordingly. How does this look now?